tests/python/gaia-ui-tests/gaiatest/apps/camera/app.py

```diff
@@ -115,6 +152,39 @@ def tap_switch_source(self):
         Wait(self.marionette).until(lambda m: not current_camera_mode == self.camera_mode)
         self.wait_for_capture_ready()
 
+    def tap_switch_to_front_camera(self):
```

Comment:
Yeah, that's a good point. If we want more contributors on the tests, we need to have PEP8 running. I filed [bug 1186388](https://bugzilla.mozilla.org/show_bug.cgi?id=1186388) for that.